content/swiftui/concepts/viewmodifier/terms/frame/frame.md

```diff
@@ -0,0 +1,103 @@
+---
+Title: '.frame()'
+Description: 'Applies an invisible frame around the View with the specified sizes.'
+Subjects:
+  - 'Mobile Development'
+  - 'iOS'
+Tags:
+  - 'SwiftUI'
+  - 'SwiftUI Modifiers'
+  - 'Xcode'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+The **`.frame()`** modifier method applies an invisible frame around the [`View`](https://www.codecademy.com/resources/docs/swiftui/views) with the specified sizes.
+
+## Syntax
+
+The `.frame()` modifier can create a fixed- or flexible-sized frame for the view. A fixed frame has a specified width and/or height. In contrast, a flexible frame can shrink and grow the view depending on the surroundings and content of the view. In both cases, the `.frame()` modifier takes various parameters for the dimensions and alignment of the created frame. Some parameters are optional and can be left out entirely or passed `nil` as a value.
+
+### Defining a fixed frame
+
+```pseudo
+Text("Some text is here.")
+    .frame(width: x, height: y, alignment: .position)
+```
+
+The `.frame()` modifier method has the following optional parameters:
+
+- `x` is entered as an integer to define the view's width in `pt` (points). SwiftUI assigns the view's width to the frame if `x` is given `nil` as a value or `width: x` is left out.
```

Comment:
Can we remove the line breaks between the bullet points.

Reply:
I removed them.